Source/Scene/Cesium3DTile.js

```diff
@@ -747,6 +768,11 @@ define([
     Cesium3DTile.prototype.visibility = function(frameState, parentVisibilityPlaneMask) {
         var cullingVolume = frameState.cullingVolume;
         var boundingVolume = getBoundingVolume(this, frameState);
+
+        if (checkTileClipped(this, boundingVolume)) {
```

Comment:
Part of this optimization is to use the original shader when the tile is not clipped.  Does this PR do that?